src/dex/fluid-dex/config.ts

```diff
@@ -0,0 +1,26 @@
+import { DexParams } from './types';
+import { DexConfigMap, AdapterMappings } from '../../types';
+import { Network, SwapSide } from '../../constants';
+
+export const FluidDexConfig: DexConfigMap<DexParams> = {
+  FluidDex: {
+    [Network.MAINNET]: {
+      pools: [
```

Comment:
these we need to fetch automatically I guess you know just mentioning to make sure

Reply:
yes DexReservesResolver returns a list of all pools. Either with reserves or without, depending on how logic for paraswap works: https://etherscan.io/address/0x278166A9B88f166EB170d55801bE1b1d1E576330#code

Reply:
otherwise it would be a maintenance nightmare to adjust config everytime when we list a new Dex pool :smiling_face_with_tear: